book/src/building/swift-package/README.md

```diff
@@ -201,9 +201,14 @@ We will need to import our rust library in `my_rust_lib.swift` and `SwiftBridgeC
 import MyRustLib
 ```
 
+<!--TODO: always make public, or maybe just have an option for making functions public in build.rs?-->
 ```swift
 // MySwiftPackage/Sources/MySwiftPackage/my_rust_lib.swift
 import MyRustLib
+
+// Also make the functions here public
+public hello_rust() -> RustString {
```

Comment:
You can make this `public func`

https://github.com/chinedufn/swift-bridge/blob/72c2fa5f2214e0845d9b3fa29b56ffbf94a97c68/crates/swift-bridge-ir/src/codegen/generate_swift.rs#L409

And then it will work